### ISSUE TITLE
yaml: Consider adding a YAML style guide

### ISSUE THREAD

**Issue by nielm** [google/styleguide#841]
Consider adding a style guide for YAML files.

Google has a YAML formatter with a certain set of default rules , which I assume expresses the style guide...

https://github.com/google/yamlfmt/blob/main/docs/config-file.md

**Comment by vapier** [comment]
we don't have a style guide.  that formatter is the only thing we have.  and even its README is clear:
> This tool is not yet officially supported by Google. It is currently maintained solely by @braydonk, and unless something changes primarily in spare time.

**Comment by braydonk** [comment]
Yes, to be clear the `yamlfmt` project doesn't make any attempt to lay out an opinionated style. The formatting is done by round-trip-ing yaml through a particular yaml parser in Go, so the default is just however that library decides to format it on output. It wasn't thought out in any way.
